### PR TITLE
[MM-819]: Added testcases for multiple functions in command_hooks.go file

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
#### Summary
Added test cases for multiple functions in command_hooks.go file

### CODE REVIEW

**Comment by wiggin77** [comment]
@Kshitij-Katiyar some CI failures created when previous PR was merged and updated here.

**Comment by Kshitij-Katiyar** [comment]
Hey @wiggin77,
I fixed the CI. It was an issue due to the duplicate function name in the same package after merging a different PR.

**Review by wiggin77 — APPROVED** [review]

File: server/command_hooks_test.go [hunk 1]
```diff
@@ -161,6 +161,29 @@ func TestExecuteCommand(t *testing.T) {
...
+		mockCluster.EXPECT().NewMutex(gomock.Eq(KeyClusterMutex)).Return(pluginapi.NewClusterMutexMock(), nil)
+		plugin, pluginContext := setupPlugin(t, mockAPI, time.Now())
+		plugin.setConfiguration(&configuration{EnableAdminRestrictions: true})
+
+		mockSystem.EXPECT().GetLicense().Return(&model.License{Features: &model.Features{
+			FutureFeatures: &trueValue,
+		}}).Times(2)
+		mockConfiguration.EXPECT().GetConfig().Return(&model.Config{}).Times(1)
+		mockUser.EXPECT().HasPermissionTo("", model.PermissionManageSystem).Return(false)
+		mockUser.EXPECT().HasPermissionToChannel("", "channel_id", model.PermissionManageChannelRoles).Return(false)
```
[the remark this answers is not in the document]
Comment (Kshitij-Katiyar) [reply server/command_hooks_test.go:175]: the mockUser which we are using to call the expected function has empty string as the userID

**Review by raghavaggarwal2308 — APPROVED** [review]

**Pull request merged**